src/metabase/middleware/exceptions.clj

```diff
@@ -39,26 +39,19 @@
   {:arglists '([e])}
   class)
 
-(defmethod api-exception-response Throwable [^Throwable e]
+(defmethod api-exception-response Throwable
+  [^Throwable e]
   (let [{:keys [status-code] :as info} (ex-data e)
 
         other-info (dissoc info :status-code :schema :type)
         message    (.getMessage e)
-        body       (cond
-                     ;; Exceptions that include a status code *and* other info are things like
-                     ;; Field validation exceptions. Return those as is
-                     (and status-code (seq other-info))
-                     (ui18n/localized-strings->strings other-info)
-
+        body       (if (and status-code (empty? other-info))
```

Comment:
should merge this tweak too -- always include extra Exception info in 500 responses